.github/workflows/build-test-release.yml

```diff
@@ -277,13 +277,72 @@ jobs:
             tests/kubernetes-distros/kind/resources/test-runs/${{ github.run_id }}
             !tests/kubernetes-distros/kind/resources/test-runs/${{ github.run_id }}/*.tgz
 
+  test-controller-for-security-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1  # We do not need the git history
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build controller image and save it to file
+        uses: docker/build-push-action@v3
+        with:
+          platforms: linux/amd64
```

Comment:
you only need to run for amd64?

Reply:
We don't run any check / tests on other platforms so I think its OK for now
Also I checked it on my arm computer and there is no difference in the issues for now